src/main/webapp/script.js

```diff
@@ -174,20 +176,22 @@ function showEvent(event, alreadyJoined) {
   }
 
   formElement.appendChild(inputElement);
-  iElement.classList.add("fas", "fa-map-marker-alt");
-  pElement.innerText = event.location;
-  pElement.appendChild(iElement);
-
+  pElement1.innerHTML = '<i class="fas fa-clock"></i> ' + event.date + ' ' + event.time;
```

Comment:
Could create element for icon so code is consistent and use innerText for date and time so there won't be any chances of injecting code

Reply:
I tried to do that but then I couldn't append things together in the order I wanted them to be. I could try to figure it out later but I wanted to have an improved UI deployed in the meantime.

Reply:
You can do it with text nodes: document.createTextNode(string) and then append the nodes in your preferred order, but you can leave it for now